Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -214,6 +214,15 @@ struct Scan {
     }
 };
 
+enum class ColorTransform {
+    // https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-T.872-201206-I!!PDF-E&type=items
+    // 6.5.3 - APP14 marker segment for colour encoding
+
+    CmykOrRgb,
```

Comment:
nit: for things from specs, I usually include the explicit `= 0` (etc) instead of relying on that being there implicitly. but up to you.

very nit: i'd also toss the newline probably